python/lsst/pipe/drivers/singleFrameDriver.py

```diff
@@ -14,11 +14,13 @@ class SingleFrameDriverTask(BatchParallelTask):
     """
     ConfigClass = SingleFrameDriverConfig
     _DefaultName = "singleFrameDriver"
+    RunnerClass = ButlerInitializedTaskRunner
 
     def __init__(self, *args, **kwargs):
+        butler = kwargs.pop("butler")
```

Comment:
This essentially makes butler a required keyword.  Unless there's a good reason not to, I'd suggest adding it to the signature `(self, *args, **kwargs)` --> `(self, butler, *args, **kwargs)`.  It makes it more obvious a butler argument is required.  But, if it can't be a positional argument, I guess we have to do it this way until Python 3.